src/Consolonia.Core/Drawing/RenderTarget.cs

```diff
@@ -166,11 +166,10 @@ private void RenderToDevice()
 
```

Comment:
The logic above this doesn't seem right:
* line 145 pixel is being clobbered with the space ? 
* line 136 pixel background should be pixel background, foreground should be invert of background. Pixel is not blended 
The sandbox page icon for HAND should look like this:
<img width="71" height="93" alt="image" src="https://github.com/user-attachments/assets/74e7af20-c609-4e6d-b174-7b6f894a6cb6" />

Instead it looks like this:
<img width="80" height="78" alt="image" src="https://github.com/user-attachments/assets/f4306e51-633d-40ed-93af-a8d3b0f7504f" />